content/docs/latest/architecture/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: SPIRE Architectural Options
+short: SPIRE Architecture
+kind: architecture
+---
+
```

Comment:
For now, it would be great to have a couple of lines explaining to a reader the broad types of problems this content can help them solve.

Reply:
I added a summary - please review.

As mentioned in a previous comment, this doesn't map to the "Build, bridge, and extend service mesh..." category on the spiffe.io home page.

Reply:
Ah, makes sense re:mapping. Summary looks great!